ReactApp/src/App.jsx

```diff
@@ -2,13 +2,15 @@
 import React from 'react';
 import RasAppCore from 'React-Automation-Studio/components/SystemComponents/RasAppCore';
 import themes  from 'React-Automation-Studio/components/UI/Themes/themes';
-import Routes from './Routes';
+import AppRoutes from "./AppRoutes";
 
 const App = (props) => {
   return (
-    <RasAppCore  themes={themes} defaultTheme={'Ocean'}   >
-      <Routes />
-    </RasAppCore>
+     <RasAppCore  themes={themes} defaultTheme={'Navy'}   >
+       <AppRoutes /> 
+     
+     </RasAppCore>
+    // <div>hello</div>
```

Comment:
Do have to keep this line? It feels like remnant of some debugging session to me.